### DISCUSSION TITLE
Prisma error with redwood js (Node > v 12.x , Yarn >= v 1.22.10, Win 7 , 32 bit )

### DISCUSSION THREAD

**Discussion by InquisitiveAboutReact** [prisma/prisma#7656] [Q&A] [answered]
Hi - This question may be a repetitive one, but facing prisma error while working with redwood js
While running the below commands , 
**yarn rw dev** & **yarn rw prisma migrate dev**  I’m getting the errors . Is it due to my system configuration? 
Any help would be appreciated. Thanks in advance !! :) 
**_yarn rw dev_**
![image](https://user-images.githubusercontent.com/85705446/121944812-b6b0bf80-cd70-11eb-996e-f373b280fa3e.png)
**_yarn rw prisma migrate dev_**
![image](https://user-images.githubusercontent.com/85705446/121944851-c3351800-cd70-11eb-960c-795e546885ab.png)

**_yarn rw dataMigrate up_**
![image](https://user-images.githubusercontent.com/85705446/121945159-17d89300-cd71-11eb-9b11-459b2f5edd0e.png)

**Comment by janpio** [comment, marked as answer]
Can you re-ask this as a bug report and submit all the information the issue template asks for? We need that for context. Thanks.

**Reply by InquisitiveAboutReact** [reply]
Hi @janpio -  I've logged the issue and as much as information provided. I am not able to remove this discussion thread. You can remove it as I've opened it under bug section. Thanks in advance ! :) 
_https://github.com/prisma/prisma/issues/7660_

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
